tests/queryDocuments/spec.ts

```diff
@@ -0,0 +1,49 @@
+import { DocumentNode, Kind, OperationTypeNode } from 'graphql';
+import { aQuery } from './mocks';
+
+it('should only return object fields present in the query document', () => {
+    const MOCKED_QUERY_DOCUMENT: DocumentNode = {
```

Comment:
You can create this object more easily using the `parse()` method provided by the `graphql` library